packages/tutorial/scripts/prepare.js

```diff
@@ -1,16 +1,29 @@
-/* global rm, cp, sed */
+/* global rm, ls, cp, sed */
 var path = require('path')
 var paths = require('./paths')
 
 function prepare (part) {
-  rm('-rf', paths.src)
-  cp('-R', path.join(paths.parts, part, 'src'), paths.root)
+  var hasSrc = ls(paths.root).reduce(function (currentHasSrc, file) {
+    if (file === 'src') {
+      return true
+    }
+
+    return currentHasSrc
+  }, false)
+
+  if (hasSrc) {
+    ls(paths.src).forEach(function (file) {
+      if (file !== 'index.js') {
+        rm('-rf', path.join(paths.src, file))
+      }
+    })
+  }
+
+  ls(path.join(paths.parts, part, 'src')).forEach(function (file) {
+    cp('-R', path.join(paths.parts, part, 'src', file), paths.src)
+  })
   cp(paths.partsIndex, paths.public)
   sed('-i', '%N%', part, paths.publicIndex)
 }
 
-if (process.argv.length > 2 && process.argv[2].indexOf('prepare.js') >= 0) {
```

Comment:
Okay, I removed package.json scripts. It is only used as a function now trough inquirer :)